packages/commonwealth/server/routes/tokenBalance.ts

```diff
@@ -66,6 +66,20 @@ const tokenBalance = async (
     }
   }
 
+  try {
```

Comment:
let's remove this whole try/catch. we have a new-and-improved version of this just above that we're currently using in production. this is redundantly old.